content/docs/reference/spec/migration/platform-api-0.6-0.7.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Platform API 0.6 -> 0.7"
++++
+
+<!--more-->
+
+This guide is most relevant to platform operators.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/platform%2Fv0.7) for platform API 0.7 for the full list of changes and further details.
+
+## Platform Operator
+
+### Run the analyzer before the detector
+
+The order of the lifecycle phases has been changed starting from Platfrom API 0.7.\
+The order before Platform API 0.7 was: detect, analyze, restore, build, and export.\
+The order starting from Platform API 0.7 is: analyze, detect, restore, build, and export.\
+The main reason for this change was to validate registry access for all images that are used prior to the `detect` and `build` phases, and therefore provide faster failures for end users.
```

Comment:
```suggestion
The main reason for this change was to validate registry access for all images that are used during the build, and therefore provide faster failures for end users.
```

Reply:
Same comment as https://github.com/buildpacks/docs/pull/412#discussion_r706395550.

Reply:
What do you think about:

The main reason for this change was to validate registry access for all images that are used during the build as early as possible, instead of during the `build` phase. Starting from Platform API 0.7, it happens in the first lifecycle phase, as part of the `analyzer`, before running both the `detector` and the `builder`, and in this way it provides faster failures for end users.

Reply:
> instead of during the `build` phase

I think in the past most of the failures would have happened in analyze and/or export, because those are the phases that need to talk to the registry. But otherwise this looks good.

Reply:
Please see https://github.com/buildpacks/docs/pull/412/commits/74d6a4cee786e2ee6473f37d70c8711759baeacb and let me know what you think.